meta/store_test.go

```diff
@@ -642,8 +673,9 @@ func TestCluster_Open(t *testing.T) {
 // Store is a test wrapper for meta.Store.
 type Store struct {
 	*meta.Store
-	Listener net.Listener
-	Stderr   bytes.Buffer
+	Listener   net.Listener
+	Stderr     bytes.Buffer
+	LeaveFiles bool // set to true to leave temporary files on close
```

Comment:
Thought about doing that but there was also the Listener that the test `Store` closes.  Someone else might add something else later that needs to be closed / cleaned up and inadvertently break this test.